spec.bs

```diff
@@ -3450,8 +3850,9 @@ Numeric value of a bid and the currency it is in.
 
 <h3 dfn-type=dfn>Generated bid</h3>
 
-The output of running a Protected Audience `generateBid()` script, which needs to be scored by
-the seller.
+A bid that needs to be scored by the seller. The bid is either the output of running a Protected
+Audience `generateBid()` script, or the additional bid provided by the "`Ad-Auction-Additional-Bid`"

```

Comment:
Link to this header properly.

Reply:
#771 has been merged now, so maybe we can do this now?

Reply:
Done. Currently put the definition under direct from seller signals section with other headers. Will clean it up by moving the fetch patching part to a common section shared by both that section and this additional bid.